tlair/src/test/scala/at/forsyte/apalache/tla/lir/IrGenerators.scala

```diff
@@ -82,8 +82,11 @@ trait IrGenerators extends TlaType1Gen {
   /**
    * Function operators (`TlaFunOper._`)
    */
-  val functionOperators = List(TlaFunOper.enum, TlaFunOper.tuple, TlaFunOper.app, TlaFunOper.domain, TlaFunOper.funDef,
-      TlaFunOper.recFunDef, TlaFunOper.recFunRef, TlaFunOper.except)
+  val functionOperators =
+    List(TlaFunOper.enum, TlaFunOper.tuple, TlaFunOper.app, TlaFunOper.domain, TlaFunOper.recFunRef, TlaFunOper.except)
+  /* TlaFunOper.{funDef,recFunDef} seem to cause lots of malformed expressions; disable them for now.
```

Comment:
So this reduces our test coverage for all things using this generator, right? Are we sure that this is the cause of the slowdown?

Reply:
Sounds good. Thanks for opening the tracking issue!